src/wire/ipv6hop.rs

```diff
@@ -0,0 +1,307 @@
+use core::fmt;
+use {Error, Result};
+
+pub use super::IpProtocol as Protocol;
+
+/// A read/write wrapper around an IPv6 Hop-by-Hop Options Header.
+#[derive(Debug, PartialEq)]
+pub struct Packet<T: AsRef<[u8]>> {
```

Comment:
`pub struct Header`, this is not a packet.

Reply:
fixed